src/components/ListOrgs.vue

```diff
@@ -431,10 +476,51 @@ const showCode = async (selectedOrg) => {
   }
 };
 
+const onEditButtonClick = (event) => {
+  isEditModalEnabled.value = true;
+  currentEditOrgId.value = _get(event, 'id', null);
+};
+
+const closeEditModal = () => {
+  isEditModalEnabled.value = false;
+  currentEditOrgId.value = null;
+};
+
 const closeDialog = () => {
   isDialogVisible.value = false;
 };
 
+const updateOrgData = async () => {
+  isSubmitting.value = true;
+  await roarfirekit.value
+    .createOrg(
+      activeOrgType.value,
+      localOrgData.value,
+      _get(localOrgData.value, 'testData', false),
+      _get(localOrgData.value, 'demoData', false),
+      currentEditOrgId.value,
+    )
+    .then(() => {
+      isSubmitting.value = false;
+      closeEditModal();
+      toast.add({
+        severity: TOAST_SEVERITIES.SUCCESS,
+        summary: 'Updated',
+        detail: 'Organization data updated successfully!',
+        life: TOAST_DEFAULT_LIFE_DURATION,
+      });
+    })
+    .catch((error) => {
+      toast.add({
+        severity: TOAST_SEVERITIES.ERROR,
+        summary: 'Unexpexcted error',
+        detail: `Unexpected error occurred: ${error.message}`,
+        life: TOAST_DEFAULT_LIFE_DURATION,
+      });
+      isSubmitting.value = false;
```

Comment:
Just a detail: to avoid the repetition `isSubmitting.value = false`, this could be moved to a final `.finally()` block